app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+
+    assert sum(split_integer(17, 4)) == 17
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+
+    assert split_integer(16, 4).count(4) == 4
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+
+    assert len(split_integer(17, 1)) == 1
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+
+    assert sorted(split_integer(32, 6)) == sorted([6, 5, 6, 5, 5, 5])
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+
+    assert split_integer(3, 4)[0] == 0 and split_integer(3, 4).count(0) == 1
```

Comment:
try to simplify it

Reply:
I try to find more harder solution, because it easy )))

Reply:
Fixed